src/main/java/unrc/dose/UserProfileService.java

```diff
@@ -0,0 +1,48 @@
+package unrc.dose;
+
+import com.google.gson.Gson;
+
+/**
+* Class which calls user profile's methods and transforms they to json.
+*/
+public final class UserProfileService {
+  /**
+  *Invokes the method which creates a user profile and transforms to json.
+  **/
+  public static String createProfile(Integer userId, String displayName, String twitterId) {
+	  
+      return UserProfile.createUserProfile(userId, displayName, twitterId).toJson(true);
+
+  }
+  /**
+   * Invokes the method which finds a user profile given a user id and transforms to json.
+   * @param userId
+   * @return
+   */
+  
+  public static String getProfile(int userId) {
+	  
+		  UserProfile up = UserProfile.findProfile(userId);
+		  return up.toJson(true, "user_id", "displayName","twitter_id");
```

Comment:
No check is done: the userId is not checked to see if a User exists with that id; and no check is done to see if there is a UserProfile for the specific userId